src/main/resources/ListedElectronicMonograph.json

```diff
@@ -75,9 +75,7 @@
         {
           "field": "DONOR",
           "dataSource": {
-            "default": "GOBI",
-            "translation": "lookupOrganization",
-            "translateDefault": true
```

Comment:
by default translateDefault is false.
Also there is no existing mappings with "translateDefault": false. If we don't need to translate - this field 'translateDefault' completely missed